From 30f491640cbea6a107c7b0b574b0d7564dcb7d5b Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Wed, 23 Oct 2024 11:14:13 +0200
Subject: [PATCH 1470/1695] plat-stm32mp2: remove deprecated shared_resource
 stub functions

Remove stm32mp2 platform shared_resources driver stub functions
stm32mp_register_secure_periph_iomem(),
stm32mp_register_non_secure_periph_iomem() and
stm32mp_register_gpioz_pin_count() since shared_resources platform driver,
inherited from stm32mp1, has been fully removed in the source tree.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 7e39348f80fd08ef9b6bae10ac6408cc46cbadb7 plat-stm32mp2: remove deprecated shared_resource stub functions]
Change-Id: I60b40258fc040e6343dd87dc607f96b513b356ad
Link: https://github.com/OP-TEE/optee_os/commit/0ba375c0d9d875041daf7bd464b05e135b0d160b
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/413528
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 core/arch/arm/plat-stm32mp2/stm32_util.h | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/core/arch/arm/plat-stm32mp2/stm32_util.h b/core/arch/arm/plat-stm32mp2/stm32_util.h
index cd236415b..e994ffe69 100644
--- a/core/arch/arm/plat-stm32mp2/stm32_util.h
+++ b/core/arch/arm/plat-stm32mp2/stm32_util.h
@@ -11,15 +11,6 @@
 #include <stdint.h>
 #include <types_ext.h>
 
-static inline void stm32mp_register_secure_periph_iomem(vaddr_t base __unused)
-{
-}
-
-static inline void stm32mp_register_non_secure_periph_iomem(vaddr_t base
-							    __unused) { }
-
-static inline void stm32mp_register_gpioz_pin_count(size_t count __unused) { }
-
 #define may_spin_lock(lock)		  cpu_spin_lock_xsave(lock)
 #define may_spin_unlock(lock, exceptions) cpu_spin_unlock_xrestore(lock, \
 								   exceptions)
-- 
2.47.2

